Parking/ParkingSpot/doc/spec.md

```diff
@@ -45,7 +45,8 @@ Thus, an entity of type `ParkingSpot` cannot exist without a containing entity o
         + `dateUpdated` : Timestamp which reflects the date when the attribute value was obtained. 
               + Type: [DateTime](https://schema.org/DateTime)
               + Optional
-        + `TimeInstant` : Timestamp saved by FIWARE's IoT Agents. Note: This attribute has not been harmonized
+        + `TimeInstant` : [Timestamp](https://github.com/telefonicaid/iotagent-node-lib/blob/develop/README.md#TimeInstant)
```

Comment:
Just confirming with you that you really want to link it to the `develop` branch of the https://github.com/telefonicaid/iotagent-node-lib project instead of `master` (where the stable, not the development version is) :)